pkgs/data/fonts/readexpro/default.nix

```
@@ -0,0 +1,49 @@
+{
+  pkgs ? import <nixpkgs> { },
+  lib ? import <nixpkgs/lib>,
+  fetchgit ? pkgs.fetchgit,
+  stdenvNoCC ? pkgs.stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "readexpro";
+  version = "1.2";
+
+  src = fetchgit {
+    url = "https://github.com/ThomasJockin/readexpro.git";
+    rev = "563dfbb36ae45e52ec50829b016ce724ac2fca70";
+    sparseCheckout = [
+      "/fonts"
+    ];
+    sha256 = "sha256-WIBbw3mvSPhbwwTX+3ntCbQn4cs7ChpuTrEujR6LVTQ=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/fonts/
+    cp -r $src/fonts/. $out/share/fonts/
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "The world-script expansion of Lexend";
```

Package descriptions shouldn't start with an article (see `pkgs/README.md` for the justification)
```suggestion
    description = "World-script expansion of Lexend";
```